src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -777,7 +805,7 @@ subroutine step_MOM_dyn_split_RK2(u, v, h, tv, visc, Time_local, dt, forces, p_s
   call cpu_clock_begin(id_clock_vertvisc)
   call vertvisc_coef(u, v, h, forces, visc, dt, G, GV, US, CS%vertvisc_CSp, CS%OBC)
   call vertvisc(u, v, h, forces, visc, dt, CS%OBC, CS%ADp, CS%CDp, G, GV, US, &
-                CS%vertvisc_CSp, CS%taux_bot, CS%tauy_bot,waves=waves)
+                CS%vertvisc_CSp, CS%taux_bot, CS%tauy_bot,waves=waves, hfrac_u=CS%diag_hfrac_u, hfrac_v=CS%diag_hfrac_v)
```

Comment:
Check to make sure that this is under 120 characters

Reply:
Sure, I will check this. Otherwise, I will add one more line.